cargo-dylint/tests/ci.rs

```diff
@@ -713,6 +713,70 @@ fn update() {
     });
 }
 
+#[test]
+fn lint_script_test() {
```

Comment:
```suggestion
fn lint() {
```
Nit. Ideally, the script will be gone, and the test suffix is redundant.